docs/docs/basics/getting-started.md

```diff
@@ -23,7 +33,14 @@ Then, you need to set up the player. This usually takes less than a second:
 ```ts
 import TrackPlayer from 'react-native-track-player';
 
-await TrackPlayer.setupPlayer()
+async function setupPlayer() {
+  await TrackPlayer.setupPlayer();
+  // The player is ready to be used
+}
+
+function App(): JSX.Element {
```

Comment:
I can see less experienced developers getting caught up on `: JSX.Element`. Perhaps they aren't using TS, just plain old JS, and they won't know what to do. I remember this happening to me in my early days.